app/src/Containers/UserForm/index.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useRef, useState } from "react";
+import { passwordValidation } from "../../Helpers/password-validation";
+import { UserFormWrapper } from "./styles";
+import { useDispatch, useSelector } from "react-redux"
+import { createUser, loginUser } from "../../actions";
+
+const creationErrorHandler = (res: any) => {
+  if(res.code === 11000){
```

Comment:
so u should be returning like some kind of object with a `success` in it that fails